Library/Formula/zorba.rb

```diff
@@ -0,0 +1,35 @@
+class Zorba < Formula
+  homepage "http://www.zorba.io/"
+  url "https://github.com/28msec/zorba/archive/3.0.tar.gz"
+  sha1 "c444cde689600aab1172b4974348dd6626e92261"
+
+  option "with-big-integer", "Use 64 bit precision instead of arbitrary precision for performance"
+  option "with-ssl-verification", "Enable SSL peer certificate verification"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "flex"
+  depends_on "icu4c"
+  depends_on "xerces-c"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    ENV["CMAKE_PREFIX_PATH"] = "#{Formula["icu4c"].prefix}:#{Formula["flex"].prefix}"
+
+    cmake_args = std_cmake_args
+    cmake_args << "-DZORBA_VERIFY_PEER_SSL_CERTIFICATE=ON" if build.with? "ssl-verification"
+    cmake_args << "-DZORBA_WITH_BIG_INTEGER=ON" if build.with? "big-integer"
```

Comment:
Did a full build with SSL enabled. Doesn't look like it taps OpenSSL anywhere, We should be good.